G/ginkgo/build_tarballs.jl

```diff
@@ -0,0 +1,57 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ginkgo"
+version = v"1.6.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/youwuyou/ginkgo.git", "c4128a21b1114bedfc19153a9508b2bd3b54954f")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/ginkgo/
+mkdir build && cd build
+cmake -DCMAKE_INSTALL_PREFIX=/workspace/destdir -DGINKGO_BUILD_TESTS=OFF -DGINKGO_BUILD_BENCHMARKS=OFF -DGINKGO_BUILD_EXAMPLES=OFF -DGINKGO_DOC_GENERATE_EXAMPLES=OFF -G "Ninja" ../
+ninja -j${nproc} 
+ninja install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl")
```

Comment:
Why this choice of platforms? The release notes suggest that it should build on Windows and MacOS as well (https://github.com/ginkgo-project/ginkgo/releases/tag/v1.6.0).

Reply:
For the rest of the platform build there are still some issues, we will try to fix that out and fix them later.

Reply:
But you basically only want the Linux platforms, right?

If so, this list can be simplified to be
```
platforms = filter!(Sys.islinux, supported_platforms())
```

Reply:
Changed to the simplified expression